handbook/engineering/web/frontend-platform/wildcard-component-library.md

```diff
@@ -0,0 +1,66 @@
+# Wildcard Component Library (WCL)
+
+Wildcard Component Library is a comprehensive collection of reusable, design-approved components that can be composed to quickly build out different user journeys across the web app. WCL is stored in its own [directory](https://github.com/sourcegraph/sourcegraph/tree/main/client/wildcard), owned by the [Fronted Platform Team](https://about.sourcegraph.com/handbook/engineering/web/frontend-platform) and contributed to by all Sourcegraphers. WCL is a part of the Wildcard Design System.
```

Comment:
> across the web app

Should we indicate here that we might want to use these reusable components outside of the web application?

Reply:
@valerybugakov could you suggest the change? Thanks!

Reply:
How about `...that can be composed to quickly build out different user journeys across our web applications.`?